packages/frontend/src/hooks/toaster/useToaster.tsx

```diff
@@ -136,6 +144,39 @@ const useToaster = () => {
         [showToast],
     );
 
+    const showToastApiError = useCallback(
+        (
+            props: Omit<NotificationData, 'subtitle'> & {
+                apiError: ApiErrorDetail;
+            },
+        ) => {
+            let title: ReactNode | undefined = props.title ?? 'Error';
+            let subtitle: ReactNode = props.apiError.id ? (
+                <p>
+                    <span>{props.apiError.message}</span>
+                    <br />
+                    <span style={{ fontWeight: 'bold' }}>
+                        Please contact support with the error ID:{' '}
+                        {props.apiError.id}
```

Comment:
Something for later: a copy-to-clipboard action icon

Reply:
Done 👍 
<img width="470" alt="Screenshot 2024-04-30 at 11 45 45" src="https://github.com/lightdash/lightdash/assets/9117144/0f324ccf-b059-44da-8d60-3debbe24fa74">